scripts/helpers.js

```diff
@@ -0,0 +1,64 @@
+import { months, directions } from "./data";
+
+export function createDOMElement(tagName, ...classNames) {
+    const element = document.createElement(tagName);
+    element.classList.add(...classNames);
+    return element;
+}
+
+export function getLocalTime(value) {
+    const time = new Date()
+    const currentUTCDate = new Date(time.getTime() + (time.getTimezoneOffset() * 60 * 1000));
+    const localTime = new Date(currentUTCDate.getTime() + (value * 1e3));
+    let hours = localTime.getHours()
+    const minutes = localTime.getMinutes().toString().padStart(2, '0')
+    const date = localTime.getDate()
+    const month = localTime.getMonth()
+    const currentMonth = months[month]
+
+    const ampm = hours >= 12 ? 'PM' : 'AM'
+    hours = hours % 12
+    hours = hours ? hours : 12;
+
+    return `${hours}:${minutes} ${ampm} ${currentMonth} ${date}`
+}
```

Comment:
blank line after the function end
```suggestion
}

```